### ISSUE TITLE
Document antipattern: Running features/scenarios in a specific order

### ISSUE THREAD

**Issue by aslakhellesoy** [cucumber/docs#696]
See https://cucumberbdd.slack.com/archives/C60TKS3SL/p1636399065115000

> I strongly recommend you avoid relying on a specific order. When a scenario fails, you should be able to run just that scenario to reproduce the same result so you can fix it. If you choose not to follow that advice, you can specify the order by listing the feature files in your runner, or as command-line arguments, depending on what Cucumber implementation you are using.

**Comment by mattwynne** [comment]
Worth referencing this: http://xunitpatterns.com/Erratic%20Test.html#Lonely%20Test

**Comment by aslakhellesoy** [comment]
Yes, but I'm worried about linking to sites with expired SSL certificates. That usually means unmaintained, which means it might go down.

**Comment by mattwynne** [comment]
Yeah shame, that's such a useful reference site. I've contacted Gerard offline to see if he can fix his SSL certs.
